docs/source/changelog.rst

```diff
@@ -5,7 +5,7 @@ Changelog
 Future Release
 ==============
     * Enhancements
-        * Add NumberOfHashtags and NumberOfMentions primitives (:pr:`180`)
+        * Add `NumberOfHashtags`, `NumberOfUniqueWords` and `NumberOfMentions`` primitives (:pr:`180`) (:pr:`187`)
```

Comment:
I'd add this PR as as separate enhancement line item, since it is unrelated to the NumberOfHashtags and NubmerOfMentions work.